app/routers/upbit_trading.py

```
@@ -0,0 +1,496 @@
+        for coin in tradable_coins:
+            currency = coin['currency']
+            market = f"KRW-{currency}"
+
+            try:
+                open_orders = await upbit.fetch_open_orders(market)
+
+                for order in open_orders:
+                    order['currency'] = currency
+                    order['korean_name'] = upbit_pairs.COIN_TO_NAME_KR.get(currency, currency)
+                    all_orders.append(order)
+            except:
```

Ruff (0.14.2)

Do not use bare `except`

(E722)